doc/release/release_0.20.rst

```diff
@@ -0,0 +1,539 @@
+  By decomposing a footprint into several smaller ones, morphological operations can potentially be sped up.
+  The decomposed footprint can be generated with the new ``decomposition`` parameter of the functions ``rectangle``, ``diamond``, ``disk``, ``cube``, ``octahedron``, ``ball``, and ``octagon`` in ``skimage.morphology``.
+  The ``footprint`` parameter of the functions ``binary_erosion``, ``binary_dilation``, ``binary_opening``, ``binary_closing``, ``erosion``, ``dilation``, ``opening``, ``closing``, ``white_tophat``, and ``black_tophat`` in ``skimage.morphology`` now accepts a sequence of 2-element tuples ``(footprint_i, num_iter_i)`` where each entry, ``i``, of the sequence contains a footprint and the number of times it should be iteratively applied. This is the form produced by the footprint decompositions mentioned above
+  (`#5482 <https://github.com/scikit-image/scikit-image/pull/5482>`_, `#6151 <https://github.com/scikit-image/scikit-image/pull/6151>`_).
+- Support anisotropic images with different voxel spacings.
+  Spacings can be defined with the new parameter ``spacing`` of the following functions in ``skimage.measure``: ``regionprops``, ``regionprops_table``, ``moments``, ``moments_central``, ``moments_normalized``, ``centroid``, ``inertia_tensor``, and ``inertia_tensor_eigvals``.
+  Voxel spacing is taken into account for the following existing properties in ``skimage.measure.regionprops``: ``area``, ``area_bbox``, ``centroid``, ``area_convex``, ``extent``, ``feret_diameter_max``, ``area_filled``, ``inertia_tensor``, ``moments``, ``moments_central``, ``moments_hu``, ``moments_normalized``, ``perimeter``, ``perimeter_crofton``, ``solidity``, ``moments_weighted_central``, and ``moments_weighted_hu``.
+  The new properties ``num_pixels`` and ``coords_scaled`` are available as well.
+  See the respective docstrings for more details
+  (`#6296 <https://github.com/scikit-image/scikit-image/pull/6296>`_).
+- Add the isotropic binary morphological operators ``isotropic_closing``, ``isotropic_dilation``, ``isotropic_erosion``, and ``isotropic_opening`` in ``skimage.morphology``.
+  These function return the same results as their non-isotropic counterparts but perform faster for large circular structuring elements
+  (`#6492 <https://github.com/scikit-image/scikit-image/pull/6492>`_).
+- Support the Modified Hausdorff Distance (MHD) metric in ``skimage.metrics.hausdorff_distance`` via the new parameter ``method``.
+  The MHD can be more robust against outliers than the directed Hausdorff Distance (HD)
+  (`#5581 <https://github.com/scikit-image/scikit-image/pull/5581>`_).
+- Added two datasets ``skimage.data.protein_transport`` and ``skimage.data.nickel_solidification``
+  (`#6087 <https://github.com/scikit-image/scikit-image/pull/6087>`_).
+- Add the new parameter ``use_gaussian_derivatives`` to ``skimage.feature.hessian_matrix`` which allows the computation of the Hessian matrix by convolving with Gaussian derivatives
```

Comment:
It feels grammatically wrong to leave out the article here. Would you prefer "Add a new parameter..." instead?

Reply:
@lagru it's like "Go to gate B." or "Film director James Cameron is famous." (you don't say: "Go to *the* gate B." nor "*The* film director James Cameron is famous.") -- I searched quickly but, unfortunately, I couldn't find a reference to point you to... With the indefinite article, it could be "Add a new parameter (namely, `use_gaussian_derivatives`) ..."

Reply:
Okay, that seems convincing. Thanks for the explanation. I'm happy to go along with the "Go to gate B" version. :smile: